extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -13,3 +13,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+amp-sidebar {
+  color: black;
+  background-color: #efefef;
+  height: 100vh;
+  --story-page-vh: 1vh;
```

Comment:
I assume we will not need to integrate with `amp-story` in `1.0` given the current versions are up for deprecation in the format.

Reply:
related: #26958

Reply:
Good idea, removed this property.